src/main.js

```diff
@@ -14,11 +19,21 @@ var currentCover;
 
 // Add your event listeners here 👇
 randomCoverButton.addEventListener("click", displayCover);
+formButton.addEventListener("click", makeYourOwnFormView);
 
 
 
 // Create your event handlers and other functions here 👇
 
+function makeYourOwnFormView() {
+  formView.classList.toggle("hidden");
+  homeView.classList.toggle("hidden");
+  randomCoverButton.classList.toggle("hidden");
```

Comment:
We included the "Make Your Own Cover" button to toggle when on the Form View page because it is not needed there and could be confusing for the user.